manifests/k/KDE/Dolphin/21.04.3/KDE.Dolphin.installer.yaml

```diff
@@ -1,16 +1,16 @@
 # Created using wingetcreate 0.4.1.1
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
 
-PackageIdentifier: KDE.Dolphin
-PackageVersion: 21.04.3
+PackageIdentifier: KDE.Dolphin.Release
+PackageVersion: master-1255
 Scope: machine
 InstallModes:
 - interactive
 - silent
 Installers:
 - Architecture: x64
   InstallerType: nullsoft
-  InstallerUrl: https://binary-factory.kde.org/job/Dolphin_Release_win64/1255/artifact/dolphin-21.04.3-1255-windows-msvc2019_64-cl.exe
+  InstallerUrl: https://binary-factory.kde.org/view/Windows%2064-bit/job/Dolphin_Release_win64/lastSuccessfulBuild/artifact/dolphin-21.04.3-1255-windows-msvc2019_64-cl.exe

```

Comment:
Why was this changed? `lastSuccessfulBuild` in the URL is only valid for 1 day, compared to having the build number in the URL which would last for about 4 or 5 days